projects/plugins/jetpack/modules/widgets/gravatar-profile.php

```diff
@@ -236,9 +236,10 @@ public function display_personal_links( $personal_links = array() ) {
 	/**
 	 * Displays the "Verified Services" accounts.
 	 *
-	 * @param array $accounts Array of social accounts.
+	 * @param array  $accounts     Array of social accounts.
+	 * @param string $display_name Display name of the user.
 	 */
-	public function display_accounts( $accounts = array() ) {
+	public function display_accounts( $accounts = array(), $display_name = '' ) {
```

Comment:
This is a `public` method, but I didn't find any other reference to this. But I added the default `''`, just in case to avoid breaking.